robocop/checkers/comments.py

```diff
@@ -129,6 +160,13 @@ def markers(self):
             self._markers = self.param("todo-in-comment", "markers").lower().split(",")
         return self._markers
 
+    @property
+    def block(self):
+        if not self._block:
+            self._block = self.param("missing-space-after-comment", "block")
```

Comment:
Since this parameter is simple and we don't need to calculate anything (like compile regex patterns), we dont need lazy loading for it. Instead of block we can use self param directly